master/security/protect-hosts.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Protect hosts
+---
+
+### Big picture
+
+Use Calico network policy to restrict traffic to/from hosts.
+
+### Value
+
+Restricting traffic between hosts and the outside world is not unique to Calico; many solutions provide this capability. However, the advantage of using Calico to protect the host is you can use the same Calico policy configuration as workloads. You only need to learn one tool. Write a cluster-wide policy, and it is immediately applied to every host. 
+
+### Features
+
+This how-to guide uses the following Calico features:
+
+- **HostEndpoint** resource 
+- **GlobalNetworkPolicy**
+- **FelixConfiguration** resource with **FailsafeInboundHostPorts** and **DefaultEndpointToHostAction** parameters
+
+### Concepts
+
+#### Hosts and workloads
+
+In the context of Calico configuration, a **workload** is a virtualized compute instance, like a VM or container. A **host** is the computer that runs the hypervisor (for VMs), or container runtime (for containers). We say it “hosts” the workloads as guests.
+
+#### Host endpoints
+
+Each host has one or more network interfaces that it uses to communicate externally. You can use Calico network policy to secure these interfaces (called host endpoints). Calico network policy for hosts can have labels, and they work the same as labels on workload endpoints. The security rules are flexible so workloads and host endpoints can refer to each other using label selectors. 
```

Comment:
> Calico network policy for hosts can have labels, and they work the same as labels on workload endpoints.

I think this should say "Calico host endpoints can have labels" rather than "Calico network policy for hosts can have labels"

Reply:
fixed.